Doppler.AccountPlans/Controllers/AccountPlansController.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Doppler.AccountPlans.DopplerSecurity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace Doppler.AccountPlans.Controllers
+{
+    [Authorize]
+    [ApiController]
+    public class HelloController
+    {
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpGet("/accounts/{accountName}/newplan/{planId}/calculate/{promocode?}")]
```

Comment:
Sinse promocode is optional, I think that it should be part of the querystring

```
 ACTUAL: https://apis.fromdoppler.com/plans/accounts/amoschini@makingsense.com/newplan/123/calculate/PROMO-123`
MY PREF: https://apis.fromdoppler.com/plans/accounts/amoschini@makingsense.com/newplan/123/calculate?promocode=PROMO-123`
```
```suggestion
        [HttpGet("/accounts/{accountName}/newplan/{planId}/calculate")]
```